guides-markdown/bcoin-clock.md

```diff
@@ -537,6 +542,90 @@ print json.dumps(getDiff(height), indent=1)
 print json.dumps(getHalf(height), indent=1)
 ```
 
+### Transaction notifications
+
+In the same way that we record `block` events from the SPV node script, we can also catch incoming transactions to the wallet, 
+and save those details to a file for the clock script to read. Back to that `(*)` section of the node script, let's add another event listener.
+We'll need to start by adding the wallet database to our node object.
+
+```javascript
```

Comment:
Since there are multiple files (of different code types), might be helpful to indicate the file name in a comment at the top of each code snippet